test/colab/test_import_login_init.ipynb

```diff
@@ -0,0 +1,70 @@
+{
+  "cells": [
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 1000
+        },
+        "id": "c2j4iPcQM_dw",
+        "outputId": "af14958b-fcb9-4a7d-c8df-8aaf64e2cd35"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install ../../dist/layer-0.10.0-py3-none-any.whl"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 461
+        },
+        "id": "vufPU6M0On4-",
+        "outputId": "1e572b8b-ce7f-4907-8c42-54b638a07789"
+      },
+      "outputs": [],
+      "source": [
+        "import layer\n",
+        "import os\n",
+        "\n",
+        "layer.login_with_api_key(os.environ.get(\"LAYER_API_KEY\"))"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "layer.init(\"test_colab\")"
```

Comment:
This has the side effect of creating a new project so it has different behaviour the first time it is run vs any other.

If we really want to create projects, then we should create a randomly named one and then delete after.


Doing just login is good enough for now though.

Reply:
I believe what happens on the backend doesn't actually matter here, as the SDK in colab will go through the same logic nonetheless?

Reply:
> I believe what happens on the backend doesn't actually matter here, as the SDK in colab will go through the same logic nonetheless?

Sorry, I don't follow what you mean. `layer.init` has a side effect as it will create a project. We should not have side effects from tests.

Reply:
Gotcha, my point what that whether the project was created or not, the behaviour on the SDK side would be the same when it comes to dependencies loaded and code path.  
I've removed that part of the test to follow your "no side effect allowed" rule.